src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -822,14 +808,15 @@ private Game GetNextWinnerGame(Game finishedGame, List<Game> games)
             }
 
             Game nextGame = games
-                .Where(g => g.GameNumber == nextGameNumber)
-                .SingleOrDefault();
+                .SingleOrDefault(g => g.GameNumber == nextGameNumber);
 
             // Check if next game can be scheduled
             ValidateEditingSchemePlayoff(nextGame);
 
             int winnerTeamId = 0;
+#pragma warning disable S3240 // The simplest possible condition syntax should be used
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)